lib/matplotlib/axes/_axes.py

```diff
@@ -3824,19 +3824,23 @@ def scatter(self, x, y, s=None, c=None, marker='o', cmap=None, norm=None,
         # Process **kwargs to handle aliases, conflicts with explicit kwargs:
 
         facecolors = None
-        ec = kwargs.pop('edgecolor', None)
+        ec = kwargs.pop('edgecolors', None)
```

Comment:
`edgecolors` is already explicitly named as a keyword argument up above. It would never be in `kwargs`.